### DISCUSSION TITLE
Does testing set need to be separated out of training folder?

### DISCUSSION THREAD

**Discussion by Mskater** [deepmodeling/deepmd-kit#751] [Q&A]
Hello,
For instance, if training the Silicon data, and "systems": are set to ["../Si/"] ,
was I supposed to separate out the testing set.009 first, so that it doesn't
train with that set?   And how do I know when it cycles to the next set in the folder?
Thanks so much for all the advice!

**Comment by njzjz** [comment]
This behavior will be deprecated in v2.0. For the version you are using, that's true.

> And how do I know when it cycles to the next set in the folder?

It's not a cycle. Every step the data will be randomly selected from all sets except the last one.
